client/post-editor/editor-seo/accordion.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import CountedTextarea from 'components/forms/counted-textarea';
+import Gridicon from 'components/gridicon';
+import InfoPopover from 'components/info-popover';
+import TokenField from 'components/token-field';
+import { getSiteSlug } from 'state/sites/selectors';
+
+class SeoAccordion extends Component {
+	constructor() {
+		super();
+
+		this.displayName = 'EditorSeoAccordion';
```

Comment:
You don't need to assign `displayName` because it's inferred from the `class`, but I think the class should probably be renamed `EditorSeoAccordion`.

Further, two things on `super`. We'd want to at least pass `props` to super, if not `...arguments` (e.g. `super( props )` or `super( ...arguments )` but once you remove the `displayName` assignment you can remove the `constructor` here altogether.